ci/jest-reporter.js

```diff
@@ -56,29 +105,40 @@ function buildDiscordStart(results, runId) {
           name: 'Started at',
           value: startedAt
         },
+        {
+          name: 'Log file (Currently all test run tasks logs running shown, will fix to one relevant one in future)',
+          value: `${logFile}`,
```

Comment:
Should this be plural? `logUrls`? In general it is helpful to pluralize if this represents more than one thing, which I think it does here?

Reply:
Updated